### PR TITLE
Remove superflous ngraph_ prefixes

### PR DESCRIPTION
**Pull request by adk9**
This reorganizes the source tree by removing many of the superflous
ngraph_ prefixes.

### CODE REVIEW

**Review by bani-intelaipg — APPROVED** [review]

File: ngraph_bridge/CMakeLists.txt [hunk 1]
```diff
@@ -30,24 +30,24 @@ message(STATUS "Shared Link Flags: ${CMAKE_SHARED_LINKER_FLAGS}")
...
+   deassign_clusters.cc
+   encapsulate_clusters.cc
    executable.cc
    ie_tensor.cc
    kernels/ngraph_encapsulate_op.cc
-   ngraph_assign_clusters.cc
+   mark_for_clustering.cc
    ngraph_builder.cc
-   ngraph_cluster_manager.cc
    ngraph_conversions.cc
```
Comment (kanvi-nervana) [inline ngraph_bridge/CMakeLists.txt:44]: Was this intentionally left out?

File: ngraph_bridge/CMakeLists.txt [hunk 2]
```diff
@@ -30,24 +30,24 @@ message(STATUS "Shared Link Flags: ${CMAKE_SHARED_LINKER_FLAGS}")
...
-   ngraph_deassign_clusters.cc
-   ngraph_encapsulate_clusters.cc
-   ngraph_mark_for_clustering.cc
    ngraph_rewrite_pass.cc
```
Comment (kanvi-nervana) [inline ngraph_bridge/CMakeLists.txt:45]: same as above

File: ngraph_bridge/kernels/ngraph_encapsulate_op.cc [hunk 3]
```diff
@@ -31,11 +31,12 @@
 
 #include "logging/ngraph_log.h"
 #include "ngraph_bridge/backend_manager.h"
+#include "ngraph_bridge/cluster_manager.h"
+#include "ngraph_bridge/ie_tensor.h"
+#include "ngraph_bridge/mark_for_clustering.h"
 #include "ngraph_bridge/ngraph_builder.h"
```
Comment (kanvi-nervana) [inline ngraph_bridge/kernels/ngraph_encapsulate_op.cc:37]: same as above

File: ngraph_bridge/CMakeLists.txt (hunk 1 above)
Comment (adk9) [reply ngraph_bridge/CMakeLists.txt:44]: Yes, I wasn't too stringent about it and left out the cases where it makes sense. The unchanged files are using the nGraph API and, for instance, it is an "nGraph builder" or those are "conversions related to nGraph".

**Review by kanvi-nervana — APPROVED** [review]

**Pull request merged**